### PR TITLE
New Contributors section

### PR DESCRIPTION
**Pull request by aayush-05**
**Screenshots**
image
---
image

_(The left indent on mobile displays is a bug on all pages, will be fixed in a separate pr)_

### CODE REVIEW

**Comment by aayush-05** [comment]
@satu0king @tachyons @nitin10s please have a look at this. Let me know if any changes required

**Comment by aayush-05** [comment]
Update : More contributors can add their name by following this simple template
`<div class="contributor-about">
                <a href="https://github.com/github_username">
                  <%= image_tag "contributors/user_image", alt: "username", class: 'contributor-image-about' %>
                </a>
              </div>`
Don't forget to add your profile name in images/contributors

**Comment by aayush-05** [comment]
It's not hardcoded @nitin10s , it basically takes up the width as other sections. For me it max up to 11 images :
image

**Comment by satu0king** [comment]
It looks great, however is there anyway to get these to load automatically from github? I am asking for a client side, like we have done here https://github.com/CircuitVerse/CircuitVerse/blob/master/CONTRIBUTING.md .

**Comment by aayush-05** [comment]
Now the contributors data are being loaded from `https://api.github.com/repos/CircuitVerse/CircuitVerse/contributors`
Let me know if any more changes are required @tachyons @nitin10s @satu0king

File: app/views/about/index.html.erb [hunk 1]
```diff
@@ -30,38 +30,32 @@
...
+            async function getContributors(url){
+              let apiData = await fetch(url);
+              let contributorsData = await apiData.json();
+              for (let i=0; i<contributorsData.length; i++){
+                let contributorElement = document.createElement('div');
+                let contributorElementAnchor = document.createElement('a');
+                let contributorElementImage = document.createElement('img');
+                contributorElement.classList.add('contributor-about');
+                contributorElementAnchor.href = await contributorsData[i]['html_url'];
+                console.log(contributorsData[i]['html_url']);
```
Comment (tachyons) [inline app/views/about/index.html.erb:51]: Can be removed

Comment (aayush-05) [reply app/views/about/index.html.erb]: Done @tachyons

**Pull request merged**